src/main/java/com/dreammaster/gthandler/GT_CraftingRecipeLoader.java

```diff
@@ -692,6 +692,76 @@ public void run() {
+
+        // Item Pipe Casings
+        // Tin
+        GT_ModHandler.addCraftingRecipe(
+                GT_ModHandler.getModItem(GregTech.ID, "gt.blockcasings11", 1, 0),
+                bits,
+                new Object[] { "PQP", "QFQ", "PQP", 'P', OrePrefixes.plate.get(Materials.Tin), 'Q',
+                        OrePrefixes.pipeMedium.get(Materials.Tin), 'F', OrePrefixes.frameGt.get(Materials.Tin) });
+
+        // Brass
+        GT_ModHandler.addCraftingRecipe(
+                GT_ModHandler.getModItem(GregTech.ID, "gt.blockcasings11", 1, 1),
+                bits,
+                new Object[] { "PQP", "QFQ", "PQP", 'P', OrePrefixes.plate.get(Materials.Brass), 'Q',
+                        OrePrefixes.pipeMedium.get(Materials.Brass), 'F', OrePrefixes.frameGt.get(Materials.Brass) });
+
+        // Electrum
+        GT_ModHandler.addCraftingRecipe(
+                GT_ModHandler.getModItem(GregTech.ID, "gt.blockcasings11", 1, 2),
+                bits,
+                new Object[] { "PQP", "QFQ", "PQP", 'P', OrePrefixes.plate.get(Materials.Electrum), 'Q',
+                        OrePrefixes.pipeMedium.get(Materials.Electrum), 'F',
+                        OrePrefixes.frameGt.get(Materials.Electrum) });
+
+        // Platinum
+        GT_ModHandler.addCraftingRecipe(
+                GT_ModHandler.getModItem(GregTech.ID, "gt.blockcasings11", 1, 3),
+                bits,
+                new Object[] { "PQP", "QFQ", "PQP", 'P', OrePrefixes.plate.get(Materials.Platinum), 'Q',
+                        OrePrefixes.pipeMedium.get(Materials.Platinum), 'F',
+                        OrePrefixes.frameGt.get(Materials.Platinum) });
+
+        // Osmium
+        GT_ModHandler.addCraftingRecipe(
+                GT_ModHandler.getModItem(GregTech.ID, "gt.blockcasings11", 1, 4),
+                bits,
+                new Object[] { "PQP", "QFQ", "PQP", 'P', OrePrefixes.plate.get(Materials.Osmium), 'Q',
+                        OrePrefixes.pipeMedium.get(Materials.Osmium), 'F', OrePrefixes.frameGt.get(Materials.Osmium) });
+
+        // Quantium
+        GT_ModHandler.addCraftingRecipe(
+                GT_ModHandler.getModItem(GregTech.ID, "gt.blockcasings11", 1, 5),
+                bits,
+                new Object[] { "PQP", "QFQ", "PQP", 'P', OrePrefixes.plate.get(Materials.Quantium), 'Q',
+                        OrePrefixes.pipeMedium.get(Materials.Quantium), 'F',
+                        OrePrefixes.frameGt.get(Materials.Quantium) });
+
+        // Fluxed Electrum
+        GT_ModHandler.addCraftingRecipe(
+                GT_ModHandler.getModItem(GregTech.ID, "gt.blockcasings11", 1, 6),
+                bits,
+                new Object[] { "PQP", "QFQ", "PQP", 'P', OrePrefixes.plate.get(Materials.ElectrumFlux), 'Q',
+                        OrePrefixes.pipeMedium.get(Materials.ElectrumFlux), 'F',
+                        OrePrefixes.frameGt.get(Materials.ElectrumFlux) });
+
+        // Black Plutronium
```

Comment:
I'm on mobile so I can't do much, but is that name correct?

Reply:
Fixed in this pr https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/950